src/software/ai/navigator/obstacle/robot_navigation_obstacle_factory.cpp

```diff
@@ -155,6 +175,7 @@ RobotNavigationObstacleFactory::createObstaclesFromMotionConstraints(
     return obstacles;
 }
 
+
```

Comment:
Super small change but maybe remove this extra line to stay consistent with spacing throughout the rest of the file.